app/src/main/java/com/github/se/assocify/ui/screens/treasury/accounting/AccountingScreen.kt

```diff
@@ -0,0 +1,156 @@
+package com.github.se.assocify.ui.screens.treasury.accounting
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.ArrowDropDown
+import androidx.compose.material3.DropdownMenu
+import androidx.compose.material3.DropdownMenuItem
+import androidx.compose.material3.FilterChip
+import androidx.compose.material3.HorizontalDivider
+import androidx.compose.material3.Icon
+import androidx.compose.material3.ListItem
+import androidx.compose.material3.ListItemDefaults
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.window.PopupProperties
+
+/**
+ * The accounting screen displaying the budget or the balance of the association
+ *
+ * @param page: The page to display (either "budget" or "balance")
+ */
+@Composable
+fun Accounting(
+    page: String,
+    categoryList: List<String>,
+    budgetLines: List<Pair<String, String>>,
+    categoryMapping: Map<String, List<String>>
+) { // TODO: fetch all these list from viewmodel
+  val yearList =
+      listOf("2023", "2022", "2021") // TODO: start from 2021 until current year (dynamically)
+  var selectedYear by remember { mutableStateOf(yearList.first()) }
+  var selectedCategory by remember { mutableStateOf(categoryList.first()) }
+
+  val filteredBudgetLines =
+      budgetLines.filter { (category, _) ->
+        categoryMapping[selectedCategory]?.contains(category) == true
```

Comment:
```suggestion
        categoryMapping[selectedCategory]?.contains(category)
```

Comparing to true is useless